manager-dashboard/app/utils/common.tsx

```diff
@@ -293,10 +285,17 @@ export const iconList: IconItem[] = [
         label: 'Swipe Left',
         component: getPngIcon(swipeIcon, 'swipe left'),
     },
+    {
+        key: 'check',
+        label: 'Check',
+        component: IoCheckmarkOutline,
+    },
```

Comment:
Let's rename and move this to iconList.

And also add check from Mapswipe app.